_specs/ecip-1095.md

```diff
@@ -0,0 +1,74 @@
+  - __Reduced risk of non-compliance__: Software agreements can have complex compliance criteria and compliance audits can be time consuming or simply a deal-breaker. Having a standardized cryptographic hash function would reduce the risk of non-compliance as _vanilla_ Sha-3 is certified by trusted organizations and has been thoroughly vetted to obtain its place in the Sha family [3].
+  - __Enhanced productivity__: Sha-3 is a solidified cryptographic function with certification and documentation. This reduces research and maintenance costs for engineers especially since Sha-3 ASIC chip designs are available.
+- __Less PoW Competition__: There are no major public blockchains whom use Sha-3 for PoW. Ethereum Classic's current market status to adopt Sha-3 would make it the majority chain for its respective PoW without having to compete in existing PoW ecosystems for miners.
+
+__Design.__
+
+As mentioned in ECIP-1049:
+
+> 1. The Proof of work function needs to be replaced with Keccak256
+> 2. The Function that checks the nonce-header in the block needs to know to accept Keccak256 hashes as valid for a block.
+
+Diagram: https://camo.githubusercontent.com/97974a2041c7633669309cf41902e4ad6b78e4cc/68747470733a2f2f692e696d6775722e636f6d2f32686f62714f4c2e706e67
+
+Instead: 
+
+1. Replace the PoW function in the EVM with _vanilla_ Sha-3-256.
+2. The function that checks the nonce-header in a block needs to accept _vanilla_ Sha-3 as valid.
+
+__Development & Testing.__
+
+Since switching to Sha-3 is resetting the mining algorithm, we need "open-source" tools and resources to help facilitate the adoption of the new PoW.
+
+- __Client implementation__: All major Ethereum Classic protocol providing clients such as Core-geth and Besu need a Sha-3 implementation.
+- __Mining-pool software__: A mining pool tool will allow anyone to contribute a mining pool to the Etherem Classic network.
+- __Mining software__: Existing AMD+NVIDIA GPU mining software for Ethash is available, but there is not solidified Sha-3 variant. We can collaborate with existing mining software providers or build new ones.
+- __Mining hardware__: We need to know the performance difference between commodity GPUs and a Sha-3 ASIC. Sha-3 is ASIC friendly and cheaper to build ASICs for than Ethash. Some chip manufacturers may be interested in participating in the initiative since is may be economically rewarding.
+- __Sha-3 testnet launch__: Launch of a Sha-3 Ethereum Classic testnet.
+- __Sha-3 network testing__: Test all operations of the testnet from mining, transactions, and smart contracts.
+
+__Implementation.__
+
+- __Recounciliation__: 
+  - Get commitment from miners and pools to contribute to the new PoW network.
+  - Have community recounciliation of testing results and make changes as needed.
+- __Set a Hard-fork date__: Assuming the Sha-3 testnet is deemed suitable for mainnet adoption, plan a hard-fork with enough time to coodinate stakeholders.
```

Comment:
I strongly believe we would need a hybrid transition period here. Having one block to change the algorithm will be madness.